doc/source/handling-dependencies.rst

```diff
@@ -0,0 +1,216 @@
+.. _handling_dependencies:
+
+Handling Dependencies
+=====================
+
+Your Ray application may depend on environment variables, files, and Python packages.
+Ray provides two features to specify these dependencies when working with a remote cluster: Runtime environments, and the Ray cluster launcher commands
+With these features, you no longer need to manually SSH into your cluster and set up your environment.
```

Comment:
I'm still not sure what an "environment" here is.

Reply:
why not replace "set up your environment" with "set up your dependencies"